### DISCUSSION TITLE
Neovim - Diagnostics Not Refreshing with Changes

### DISCUSSION THREAD

**Discussion by TomMalitz** [vuejs/language-tools#1918] [General]
I'm currently using Neovim with Volar (LSP setup using [lsp-zero](https://github.com/VonHeikemen/lsp-zero.nvim)). 

I noticed that any changes I make to .ts files within my project are not dynamically picked up by Volar. In order to get proper diagnostics I need to restart the LSP manually. Is this expected behavior?

I am new to Neovim in general as well as configuring LSP settings. I'm assuming my issue stems from using lsp-zero and not configuring Volar manually but I would appreciate any input on how to get my diagnositcs to properly refresh as changes are made to other files.

Thanks!

**Comment by johnsoncodehk** [comment]
Sorry I have no experience with Neovim so I don't know, but maybe related to: https://github.com/yaegassy/coc-volar#recommended-additional-installation-watchman
